bindings/node/lib/autoEncrypter.js

```diff
@@ -321,7 +319,9 @@ module.exports = function (modules) {
      * the original ones.
      */
     async askForKMSCredentials() {
-      return this._onKmsProviderRefresh ? this._onKmsProviderRefresh() : {};
+      return this._onKmsProviderRefresh
```

Comment:
If the user provides a function we still need to use the deprecated option, but if they do not we refresh from the env.